home-page.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+<link rel="stylesheet" href="home-page-style.css">
+</head>
+<body>
+  <header class="header">
+<div id="recipe-page">Recipe</div>
+<div id="create-page">Create new Recipe</div>
+
+  </header>
+  <div class="search-bar">
+  <input type="text" id="search-input" placeholder="Search for recipe">
+  <button id="search-button">Search</button>
```

Comment:
Always add the button type attribute, as some browsers interpret it by default as `submit`, and others as `button` which can result in inconsistent behaviors. 
```suggestion
  <button id="search-button" type="button">Search</button>
```